docs/content/docs/testing/telemetry.md

```diff
@@ -240,24 +236,55 @@ export class ConsoleLogger implements ITelemetryBaseLogger {
-    // and hook to the Telemetry system
-    const consoleLogger: ConsoleLogger = new ConsoleLogger();
-
-    // Get or create the document depending if we are running through the create new flow
-    const client = useAzure ? AzureClient :  new TinyliciousClient();
-    const [fluidContainer, containerServices] = createNew
-        ? await client.createContainer({ id: containerId, logger: consoleLogger }, containerSchema)
-        : await client.getContainer({ id: containerId, logger: consoleLogger }, containerSchema);
+  // Create a custom ITelemetryBaseLogger object to pass into the Tinylicious container
+  // and hook to the Telemetry system
+  const azureConfig = {
+      connection: connectionProps,
+      logger: new ConsoleLogger(),
+  };
+  const client = new AzureClient(azureConfig);
+  let container: FluidContainer;
+  let services: AzureContainerServices;
+
+  // Get or create the document depending if we are running through the create new flow
+  const createNew = !location.hash;
+  if (createNew) {
+      // The client will create a new detached container using the schema
+      // A detached container will enable the app to modify the container before attaching it to the client
+      {container, services} = await client.createContainer(containerSchema);
+  }
 ```
 
 Now, whenever a telemetry event is encountered, the custom `send()` method gets called and will print out the entire
 event object.
 
 <img src="https://fluidframework.blob.core.windows.net/static/images/consoleLogger_telemetry_in_action.png" alt="The
   ConsoleLogger sends telemetry events to the browser console for display.">
+
+{{% callout warning %}}
+
+The purpose of `ConsoleLogger` is purely to demonstrate how the `ITelemetryBaseLogger` interface should be implemented. The
+Fluid Framework provides `DebugLogger` by default if a custom logger is not provided. For more information, please refer to
+[Using DebugLogger](#using-debuglogger) below instead of implementing something similar to `ConsoleLogger`.
+
+{{% /callout %}}
+
+### Using `DebugLogger`
+
+The `DebugLogger` offers a convenient way to output all telemetry events to the console. Like mentioned previously, creating
+or getting a container creates a `Loader` object, which then also provides a `DebugLogger` as a mixin in its constructor. Hence,
+`DebugLogger` is already present by default when creating/getting a container, and no extra steps are required to use it.
+
+Under the hood, `DebugLogger` uses the [debug](https://github.com/visionmedia/debug) library. `Debug` allows a library to log
+messages to a namespace. By default these messages aren't displayed but can be enabled by the app that is using the library. For
+our node apps enabling library logging can be done by setting the DEBUG environment variable - i.e.
+
+In the browser you can enable them by setting `localStorage.debug` variable, after which you will need to reload the page.
+
+`localStorage.debug = 'fluid:*'`
+
```

Comment:
```suggestion
```suggestion
Under the hood, `DebugLogger` uses the [debug](https://github.com/visionmedia/debug) library. The `debug` library enables Fluid to send to a unique 'namespace,' `fluid`. By default these messages are hidden but they can be enabled
in both Node.js and a web browser.

**To enable Fluid Framework logging in the browser,** set the `localStorage.debug` variable in the JavaScript console, after which you will need to reload the page.

\`\`\`js
localStorage.debug = 'fluid:*'
\`\`\`

**To enable Fluid Framework logging in a Node.js application,** set the `DEBUG` environment variable when running the app.

```

I think you're missing the code sample for node.